### PR TITLE
Refactoring to conventional view structure

### PR DESCRIPTION
**Pull request by monokrome**
Refactoring to conventional view structure

### CODE REVIEW

File: fighthealthinsurance/views.py [hunk 1]
```diff
@@ -451,64 +447,80 @@ def ocr_upload(x):
...
+        # but we don't support changing the values.
+        email = form.cleaned_data["email"]
+        hashed_email = Denial.get_hashed_email(email)
+        denial_text = form.cleaned_data["denial_text"]
+
+        denial = Denial.objects.create(
+            denial_text=denial_text,
+            hashed_email=hashed_email,
+        )
+        denial.save()
```
Comment (monokrome) [inline fighthealthinsurance/views.py]: I think this was added, but we don't need to save here unless the **Denial** Model has some magic in its save method or (possibly?) a *pre_save* / *post_save* hook. The `create()` makes a database record itself.

File: fighthealthinsurance/views.py [hunk 2]
```diff
@@ -493,7 +494,8 @@ def form_valid(self, form):
         state = None
         zip_code = form.cleaned_data["zip"]
         if zip_code is not None and zip_code != "":
-            state = self.zip_engine.by_zip_code(form.cleaned_data["zip"]).state
+            print(f"Zip '{zip_code}'")
```
Comment (monokrome) [inline fighthealthinsurance/views.py]: Accidentally a print? 🙃

File: fighthealthinsurance/views.py (hunk 1 above)
Comment (holdenk) [reply fighthealthinsurance/views.py]: Oh yeah good point, I put it in while I was trying to figure out why it was not working but it was the fixtures.

File: fighthealthinsurance/views.py (hunk 2 above)
Comment (holdenk) [reply fighthealthinsurance/views.py]: Oh yeah was debugging that too :p lets drop it

**Pull request merged**